build.sh

```diff
@@ -508,6 +512,40 @@ build_hhvm() {
 }
 
 
+# autoconf-2.13 is needed to build spidermonkey
+build_autoconf() {
+    echo "\n===> Download and build autoconf-2.13\n"
+    if [ -d ${wrkdir}/autoconf-2.13 ]; then return; fi
+    cd ${wrkdir} || exit $?
+    wget http://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz || exit $?
+    tar xfz autoconf-2.13.tar.gz || exit $?
+    cd autoconf-2.13
+    ./configure --prefix=${wrkdir}/autoconf-inst || exit $?
+    make install || exit $?
+    cd ${wrkdir}/autoconf-inst/bin
+    ln -s autoconf autoconf-2.13 || exit $?
+    cd ${HERE}
+}
+
+
+SPIDERMONKEY_VERSION=465d150bc8be # FIREFOX_AURORA_50_BASE
+build_spidermonkey() {
+    echo "\n===> Download and build SpiderMonkey\n"
+    if [ -d ${wrkdir}/spidermonkey ]; then return; fi
+    cd ${wrkdir} || exit $?
+    wget -O spidermonkey.tar.bz2 http://hg.mozilla.org/mozilla-central/archive/${SPIDERMONKEY_VERSION}.tar.bz2 || exit $?
+    bunzip2 -c - spidermonkey.tar.bz2 | tar xfp - || exit $?
```

Comment:
I'd prefer to stick to the known portable route, rather than being a bit clever, and then finding out down the line that it's not always supported.